### DISCUSSION TITLE
How to move forward from discussion to Proposal? Or next step in moving forward?

### DISCUSSION THREAD

**Discussion by ne0rrmatrix** [CommunityToolkit/Maui#1143] [Q&A] [answered]
A few weeks ago I started a discussion with a feature I wanted to add to media element. It is full screen support for android and windows. I have heard that it a feature that has been asked for and at least on maintainer has shown support by saying it was feature he was aware the community had asked for.

I am unsure how to move this forward. I have created a discussion about it and have had zero response. Am I being inpatient? Is my discussion not fleshed out enough for anyone to comment? Have I formatted it incorrectly? Is it incomplete? Or is there more I need to do before the next step? Or am I missing something that I need to do? I have read the contribution guide and from that I gathered I wait for one of the maintainers to move it forward?

I am curious as I have done some work by forking the repo and doing the initial work to add the feature. I am hoping I am simply missing something or that I am just impatient and have not made some sort of error in how I submitted the discussion. Or am I missing something that is simple that I overlooked?

Can someone look at this discussion: [https://github.com/CommunityToolkit/Maui/discussions/1123](https://github.com/CommunityToolkit/Maui/discussions/1123) and give me some feedback about how it is formatted and if I need to make changes. I would also like to know if I am simply being impatient or missed something I should be doing? Or is it something else?

**Comment by EdwardMcFlurry** [comment]
@ne0rrmatrix you are definitely not impatient but you are doing exactly what any Developer would do which is to ask for further assistance.

I’m also curious about Full screen feature in Media Element and I hope someone can help you move forward.

**Comment by pictos** [comment, marked as answer]
@ne0rrmatrix thanks for creating the discussion around the feature... Sometimes we take longer to look into them since the core team works on this lib during their free time. Sometimes we don't have the time to look into all proposals, bugs, code reviews, and PR and we should move to what's the high priority at the moment (usually bug fixes). So, don't worry we will take a look at your proposal ASAP.
